docs/source/federation-guide/graph-administration.mdx

```diff
@@ -0,0 +1,201 @@
+
+## Observability
+
+As more services are added to a federated data graph and adoption spreads across client applications, it may grow challenging to reason about how a single request traverses the graph. Apollo Gateway undertakes a query planning process to optimize for the most time spent in a single service to reduce the number of network hops for a single request. Once calculated, the gateway will execute the query plan across the subgraphs required to fulfill the request. With managed federation and Apollo Studio, [federated traces](https://www.apollographql.com/docs/federation/metrics/#gatsby-focus-wrapper) may be used to provide detailed insights into the GraphQL layer's performance and usage.
+
+### Federated Traces
+
+With federated tracing enabled ([which happens by default](https://www.apollographql.com/docs/federation/metrics/#turning-it-on) when an `APOLLO_KEY` variable is present in the gateway's environment), the gateway will include an HTTP header of `apollo-federation-include-trace: ftv1` with each request to a subgraph. Each subgraph will then [construct its trace](https://www.apollographql.com/docs/federation/metrics/#how-traces-are-constructed-and-aggregated) and add this data to the `extensions` of its response. Apollo Gateway then constructs the overall trace for the request based on the shape of the query plan. This process may be visualized as follows:
+
+![Federated tracing process](../images/federation-guide/federated-tracing.png)
+
+The gateway will send this tracing data to Apollo Studio where it may be used for schema checks, tuning query performance, and debugging operation errors:
+
+![A federated trace representation in Apollo Studio](../images/federation-guide/apollo-studio-federated-trace.png)
+
+When Apollo Server is used to power subgraphs, this tracing data is provided out-of-the-box via the [inline trace plugin](https://www.apollographql.com/docs/apollo-server/api/plugin/inline-trace/#using-the-plugin). Many [third-party federation libraries](https://www.apollographql.com/docs/federation/other-servers/) also expose federated tracing data. Note that not all third-party federation libraries will necessarily provide field-level tracing data in a response, but the gateway's aggregated trace will still show the total time spent in the service even though the detailed field resolver data won't be available for that portion of the query plan.
+
+### Integration with Other Observability Tools
```

Comment:
@prasek should we put in OpenTelemetry in this? If the releases line up for the end of the month that would be super nice

Reply:
@michael-watson The plan was to release this version of the guide asap, so I can do a later PR to incorporate the OTel-related updates in this section at the direction of @prasek

Reply:
@mandiwise @michael-watson fast-follow for Otel sounds good 👍